### PR TITLE
C9400 series chassis

### PR DESCRIPTION
**Pull request by arcticash**
Updating the Chassis Switch (9400) model to use Modules for Line Cards & PSU's as they are removable and swappable.

Also adding and updating the Linecard, PSU & Supervisor modules

### CODE REVIEW

File: device-types/Cisco/C9407R.yaml [hunk 1]
```diff
@@ -3,19 +3,55 @@ manufacturer: Cisco
...
-    maximum_draw: 3200
-interfaces:
-  - name: GigabitEthernet0/0
-    type: 1000base-t
-    mgmt_only: true
+weight: 28.60
+weight_unit: kg
+is_full_depth: true
+airflow: right-to-left
+module-bays:
```
Comment (danner26) [inline device-types/Cisco/C9407R.yaml:12]: Does the device itself not have a management port on it? It appears that the old definiction has a single interface, GigabitEthernet0/0 which was management only. It also appears to have has a console port as well. Are these not in the chassis by default?

I am also asking @DanSheps for further clarification

File: module-types/Cisco/C9400-SUP-1XL-Y.yaml [hunk 2]
```diff
@@ -28,3 +33,6 @@ interfaces:
     type: 25gbase-x-sfp28
   - name: TwentyFiveGigE{module}/0/5
```
Comment (danner26) [inline module-types/Cisco/C9400-SUP-1XL-Y.yaml:34]: I see the interface before this one is `TwentyFiveGigE{module}/0/1`, and then it goes to `TwentyFiveGigE{module}/0/5`... Is that correct or should this be `TwentyFiveGigE{module}/0/2`?

File: device-types/Cisco/C9404R.yaml [hunk 3]
```diff
@@ -3,19 +3,34 @@ manufacturer: Cisco
...
-    maximum_draw: 3200
-interfaces:
-  - name: GigabitEthernet0/0
-    type: 1000base-t
-    mgmt_only: true
+weight: 17.21
+weight_unit: kg
+is_full_depth: true
+airflow: right-to-left
+module-bays:
```
[the remark this answers is not in the document]
Comment (danner26) [reply device-types/Cisco/C9404R.yaml:12]: Talked with Dan, he said they are on the supervisor. You can ignore this comment.

File: device-types/Cisco/C9410R.yaml [hunk 4]
```diff
@@ -3,19 +3,64 @@ manufacturer: Cisco
...
-    maximum_draw: 3190
-  - name: PS-2
-    type: iec-60320-c14
-    maximum_draw: 3190
-interfaces:
-  - name: GigabitEthernet0/0
-    type: 1000base-t
-    mgmt_only: true
+airflow: right-to-left
+module-bays:
```
[the remark this answers is not in the document]
Comment (danner26) [reply device-types/Cisco/C9410R.yaml:12]: Talked with Dan, he said they are on the supervisor. You can ignore this comment.

File: device-types/Cisco/C9407R.yaml (hunk 1 above)
Comment (danner26) [reply device-types/Cisco/C9407R.yaml:12]: Talked with Dan, he said they are on the supervisor. You can ignore this comment.

File: module-types/Cisco/C9400-SUP-1XL-Y.yaml (hunk 2 above)
Comment (arcticash) [reply module-types/Cisco/C9400-SUP-1XL-Y.yaml]: Now fixed in [f102fb1](https://github.com/netbox-community/devicetype-library/pull/1329/commits/f102fb180ec96a0e0586af4d6266c29062a947b3)

**Pull request merged**